tools/sysadmin-vm-prep.sh

```
@@ -0,0 +1,138 @@
+    echo -e "${RED}❌ Error: No target host provided.${NC}"
+    echo -e "${YELLOW}Usage: bash tools/sysadmin-vm-prep.sh <ip-address-or-hostname>${NC}"
+    exit 1
+fi
+
+echo -e "${CYAN}==================================================${NC}"
+echo -e "${CYAN}    PROVISIONER: BOOTSTRAPPING REMOTE VM          ${NC}"
+echo -e "${CYAN}    Target: $TARGET_HOST${NC}"
+echo -e "${CYAN}==================================================${NC}"
+
+# Interactive Credential Collection
+read -p "👤 Enter Initial SSH Username (e.g., admin, root, user): " REMOTE_USER
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)